crates/topos/src/components/mod.rs

```diff
@@ -1,13 +1,3 @@
-pub(crate) mod doctor;
-#[cfg(feature = "network")]
-pub(crate) mod network;
-#[cfg(feature = "node")]
 pub(crate) mod node;
-#[cfg(feature = "sequencer")]
-pub(crate) mod sequencer;
-#[cfg(feature = "setup")]
+pub(crate) mod regtest;
```

Comment:
`regtest` isn't clear for me :/ 
I don't understand what's being this

Reply:
It's just naming? We can call it utils, or whatever! We have two commands related to the testnet and/or local setup! 

Other projects use the regtest name for this. But we can choose whatever!

Or what isn't clear?